AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/routes/v1/PictureRoutes.java

```diff
@@ -80,4 +87,19 @@ public Object deletePicture(Request req, Response res, int id) throws Exception
 
         return new Object();
     }
+
+    private ArrayList<String> getAllowedContentTypes(){
+        String[] allowedContent = configuration.getStringArray("database.picture.allowed.content.types");
```

Comment:
In my mind the dots denote a hierarchy, rather than simply separating words. It's all a matter of taste, of course.

Perhaps `database.picture.allowed-content-types`?